tests/new_upgrades/test_capsule.py

```diff
@@ -0,0 +1,267 @@
+
+    :steps:
+        1. Before Satellite upgrade, sync a RPM repo to Satellite
+
+    :expectedresults:
+        1. The repo/rpm should be synced to satellite
+        2. Activation key's environment id should be available in the content views environment
+            id's list
+    """
+    target_sat = capsule_upgrade_integrated_sat_cap.satellite
+    capsule = capsule_upgrade_integrated_sat_cap.capsule
+    cap_smart_proxy = capsule_upgrade_integrated_sat_cap.cap_smart_proxy
+    with (
+        SharedResource(target_sat.hostname, upgrade_action, target_sat=target_sat) as sat_upgrade,
+        SharedResource(capsule.hostname, upgrade_action, target_sat=capsule) as cap_upgrade,
+    ):
+        test_name = f'capsule_sync_{gen_alpha()}'
+        org = target_sat.api.Organization(name=f'{test_name}_org').create()
+        cap_smart_proxy.organization = [org]
+        cap_smart_proxy.update(['organization'])
+        features = json.loads(capsule.get_features())
+        prior_lce_id = int(
+            target_sat.cli.LifecycleEnvironment.list(
+                {
+                    'organization-id': org.id,
+                    'search': 'Library',
+                }
+            )[0]['id']
+        )
+        lce = target_sat.api.LifecycleEnvironment(
+            name=f'{test_name}_lce', organization=org, prior=prior_lce_id
+        ).create()
+        capsule.nailgun_capsule.content_add_lifecycle_environment(data={'environment_id': lce.id})
+        product = target_sat.api.Product(name=f'{test_name}_prod', organization=org).create()
+        repo = target_sat.api.Repository(
+            name=f'{test_name}_repo',
+            product=product,
+            content_type='yum',
+            url=settings.repos.yum_1.url,
+        ).create()
+        repo.sync()
+        content_view = target_sat.publish_content_view(org, repo, f'{test_name}_content_view')
+        content_view.version[0].promote(data={'environment_ids': [lce.id]})
+        content_view_env_id = [env.id for env in content_view.read().environment]
+        ak = target_sat.api.ActivationKey(
+            name=f'{test_name}_ak', organization=org.id, environment=lce, content_view=content_view
+        ).create()
+        assert lce.id in content_view_env_id
+        test_data = Box(
+            {
+                'target_sat': target_sat,
+                'capsule': capsule,
+                'features': features,
+                'org': org,
+                'lce': lce,
+                'product': product,
+                'repo': repo,
+                'content_view': content_view,
+                'ak': ak,
+            }
+        )
+        sat_upgrade.ready()
+        target_sat._swap_nailgun(f"{settings.UPGRADE.TO_VERSION}.z")
+        cap_upgrade.ready()
+        target_sat._session = None
+        capsule._session = None
+        yield test_data
+
+
+@pytest.mark.capsule_upgrades
+def test_post_capsule_features(capsule_sync_setup):
```

Comment:
I get that currently we want new test names to get mapped to old ones.

But going forwards I don't think there has to be word `_post` in all test names